window.go

```diff
@@ -54,3 +54,7 @@ func GetWindowThreadID(hwnd syscall.Handle) (uint, error) {
 	}
 	return uint(r), nil
 }
+
+func RegisterWindowMessage(messageName string) (uint32, error) {
+	return wrappers.RegisterWindowMessage(syscall.StringToUTF16Ptr(messageName))
```

Comment:
Should wrap the error with `NewWindowsError` here.  Also, let's define a new data type `WindowMessage` to hold a message so that we have some type-safety.